@navikt/core/css/darkside/timeline.darkside.css

```diff
@@ -320,22 +350,14 @@
 
 @supports not selector(:focus-visible) {
```

Comment:
We have for the most part removed checks for `not selector(:focus-visible)`. This component is mostly (only?) used by internal applications that require modern browsers.